src/main/java/com/squareup/kotlinpoet/Util.kt

```diff
@@ -127,6 +127,9 @@ internal fun stringLiteralWithQuotes(value: String): String {
   }
 }
 
+internal fun escapeKeywords(canonicalName: String) =
+        canonicalName.split('.').joinToString(".") { escapeIfKeyword(it) }
```

Comment:
We use 2-space indents (4-space for continuation indents), could you please format your code accordingly?